emmet-api/setup.py

```diff
@@ -8,9 +8,9 @@
 setup(
     name="emmet-api",
     use_scm_version={
-        "root": ".",
+        "root": "..",
```

Comment:
This basically reverts https://github.com/materialsproject/emmet/pull/565

Reply:
Yes, I had to revert as the git information wasn't properly being accessed by setuptools when root is set to `.`

I will try and find another solution, but in the mean time this change needed to happen so our deployment pipelines don't break.